federation/pkg/federation-controller/ingress/ingress_controller_test.go

```diff
@@ -40,7 +40,20 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	testIterations = 1
+)
+
 func TestIngressController(t *testing.T) {
+	for i := 0; i < testIterations; i++ {
```

Comment:
We need this. We should ask people to run a 100 test runs when modifying anything in the controllers, so that flakes are spotted earlier.